akka-http/src/main/scala/akka/http/scaladsl/server/directives/FormFieldDirectives.scala

```diff
@@ -52,22 +55,35 @@ trait FormFieldDirectives extends ToNameReceptacleEnhancements {
    *
    * @group form
    */
-  def formField(pdm: FieldMagnet): pdm.Out = pdm()
+  override def formField(pdm: FieldMagnet): Directive[pdm.U] = formFields(pdm)
 
   /**
    * Extracts a number of HTTP form field from the request.
    * Rejects the request if the defined form field matcher(s) don't match.
    *
    * @group form
    */
-  def formFields(pdm: FieldMagnet): pdm.Out = pdm()
+  override def formFields(pdm: FieldMagnet): Directive[pdm.U] =
+    toStrictEntity(StrictForm.toStrictTimeout).wrap { pdm() }
```

Comment:
That's basically the fix. We want to be sure that the complete inner part of a formField directive is wrapped in a `toStrictEntity` directive.

The problem in the old version was, that we couldn't prove that `pdm()` actually is a directive, so we couldn't use `wrap`.